central/resourcecollection/service/service_impl.go

```diff
@@ -166,7 +166,7 @@ func (s *serviceImpl) DeleteCollection(ctx context.Context, request *v1.Resource
 		return nil, errors.Wrap(err, "Failed to check for Report Configuration usages")
 	}
 	if reportConfigCount != 0 {
-		return nil, errors.Wrap(errox.ReferencedByAnotherObject, "Collection is in use by one or more report configurations")
+		return nil, errors.Wrap(errox.ReferencedByAnotherObject, "Failed to delete collection")
```

Comment:
why not update the test? https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/vulnmanagement/reporting/reportingForm.test.js#L239